idl/chromadb/proto/coordinator.proto

```diff
@@ -64,6 +66,7 @@ message GetSegmentsRequest {
   optional string id = 1;
   optional string type = 2;
   optional SegmentScope scope = 3;
+  // required
   optional string collection = 5; // Collection ID
```

Comment:
why jump to field 5?

Reply:
this was already here, I just added the comment above

Reply:
hm, maybe we should just change to 4 since no one would be affected